frontend/src/components/hints/Hints.jsx

```diff
@@ -0,0 +1,50 @@
+import "./hints.css";
+import PropTypes from "prop-types";
+import parisLocationHints from "../../data/parisLocationHints.json";
+
+function Hints({ pictureAnatorParis }) {
+  // je récupère l'index du tableau parisLocationHints avec pictureAnatorParis
+  // je push les 5 hints sur un nouveau tableau
+  const newHintsArray = [
+    pictureAnatorParis.hint1,
+    pictureAnatorParis.hint2,
+    pictureAnatorParis.hint3,
+    pictureAnatorParis.hint4,
+    pictureAnatorParis.hint5,
+  ];
+  let randomIndex = "";
+  const arrayHintstoDisplay = [];
+  const gettingRandomNumberupTo5 = (max) => {
+    randomIndex = Math.floor(Math.random() * max);
+  };
+  // merci Ayoub pour l'aide ^^
+  const indexNumbers = new Set();
+  while (indexNumbers.size < 3) {
+    gettingRandomNumberupTo5(5);
+    indexNumbers.add(randomIndex);
+  }
+  const newArray = [...indexNumbers];
+  for (const index of newArray) {
+    arrayHintstoDisplay.push(newHintsArray[index]);
+  }
+  // pour l'instant on code en dur les hints mais on pourra dans le futur les rendre plus dynamiques avec des states en fonction du click bouton
```

Comment:
You can remove this comment and add a map to display your hints

Reply:
I've removed the comment but I'll refrain from "maping" the array as I'll write some code later (maybe today) using a state and a function onClick to display them while clicking on buttons.